content/2017/12/15/figma-trends.rst

```diff
@@ -0,0 +1,17 @@
+public: yes
+author: miriam
+canonical: https://www.figma.com/blog/eighteen-designers-predict-ui-ux-trends-for-2018/
+elsewhere: Figma
+tags: [Elsewhere, Design, CSS, Interview, Figma]
+image:
+  - src: 'blog/2017/figma-trends.jpg'
+summary: |
+  Figma asked 18 designers (including me)
+  what UI/UX trends they predict for 2018.
+  December, with its flurry of holiday parties,
+  cheery OOO auto-replies and introspective weather patterns,
+  provided the perfect opportunity to pause and survey the landscape.
+
+
+Figma: 18 designers predict UI/UX trends for 2018
```

Comment:
Are we keeping headlines Title Case still? If so, maybe we can update this? 
https://deploy-preview-359--oddsite.netlify.com/styleguide/#headlines